week3.ipynb

```diff
@@ -0,0 +1,495 @@
+       "4             5      Boron      B      10.811                 6   \n",
+       "\n",
+       "   NumberofProtons  NumberofElectrons  Period  Group  Phase  ...  \\\n",
+       "0                1                  1       1    1.0    gas  ...   \n",
+       "1                2                  2       1   18.0    gas  ...   \n",
+       "2                3                  3       2    1.0  solid  ...   \n",
+       "3                4                  4       2    2.0  solid  ...   \n",
+       "4                5                  5       2   13.0  solid  ...   \n",
+       "\n",
+       "  FirstIonization   Density MeltingPoint BoilingPoint NumberOfIsotopes  \\\n",
+       "0         13.5984  0.000090       14.175        20.28              3.0   \n",
+       "1         24.5874  0.000179          NaN         4.22              5.0   \n",
+       "2          5.3917  0.534000      453.850      1615.00              5.0   \n",
+       "3          9.3227  1.850000     1560.150      2742.00              6.0   \n",
+       "4          8.2980  2.340000     2573.150      4200.00              6.0   \n",
+       "\n",
+       "   Discoverer    Year  SpecificHeat  NumberofShells  NumberofValence  \n",
+       "0   Cavendish  1766.0        14.304               1              1.0  \n",
+       "1     Janssen  1868.0         5.193               1              NaN  \n",
+       "2   Arfvedson  1817.0         3.582               2              1.0  \n",
+       "3  Vaulquelin  1798.0         1.825               2              2.0  \n",
+       "4  Gay-Lussac  1808.0         1.026               2              3.0  \n",
+       "\n",
+       "[5 rows x 28 columns]"
+      ]
+     },
+     "execution_count": 36,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# read in our csv file\n",
+    "periodic_table = pd.read_csv('data/Periodic Table of Elements.csv')\n",
+    "\n",
+    "# check it out\n",
+    "periodic_table.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "27\n",
+      "1\n"
+     ]
+    }
+   ],
+   "source": [
+    "# we'll convert it to a numpy array for simplicity\n",
+    "periodic_table_array = periodic_table.to_numpy()\n",
+    "\n",
+    "# we can still use our dataframe to find which columns we want to index in our numpy array\n",
+    "print(periodic_table.columns.get_loc('NumberofValence'))\n",
+    "print(periodic_table.columns.get_loc('Element'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# let's use a for loop in combination with if-else statements to pull out a class of elements\n",
```

Comment:
For readability, I would recommend putting multi-line comments like this outside of your code snippet.